docs/content/reference/install-configuration/providers/others/http.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Traefik HTTP Documentation"
+description: "Provide your dynamic configuration via an HTTP(S) endpoint and let Traefik Proxy do the rest. Read the technical documentation."
+---
+
+# Traefik & HTTP
+
+Provide your [install configuration](../overview.md) via an HTTP(S) endpoint and let Traefik do the rest!
+
+## Configuration Example
+
+You can enable the HTTP provider as detailed below:
```

Comment:
For the HTTP provider the endpoint option is required so we should use it in the following examples (it does not have a default value).

See https://doc.traefik.io/traefik/providers/http/#endpoint